xml/bp_troubleshooting_cephfs.xml

```diff
@@ -87,132 +87,6 @@ ceph daemon mds.<replaceable>NAME</replaceable> dump_ops_in_flight
      </para>
    </sect2>
  </sect1>
- <sect1 xml:id="bp-troubleshooting-cephfuse-debugging">
-   <title>Ceph-Fuse Debugging</title>
-   <para>
-     <command>ceph-fuse</command> also supports <option>dump_ops_in_flight</option>.
-     See if it has any and where they are stuck.
-   </para>
-   <sect2 xml:id="debug-output">
-     <title>Debug Output</title>
-     <para>
-       To get more debugging information from <command>ceph-fuse</command>, try running
-       in the foreground with logging to the console (<option>-d</option>) and enabling
-       client debug (<option>--debug-client=20</option>), enabling prints for each
-       message sent (<option>--debug-ms=1</option>).
-     </para>
-     <para>
-       If you suspect a potential monitor issue, enable monitor
-       debugging as well (<option>--debug-monc=20</option>).
-     </para>
-   </sect2>
- </sect1>
- <sect1 xml:id="bp-troubleshooting-kernelmount-debugging">
-   <title>Kernel Mount Debugging</title>
```

Comment:
Just checking: is this being removed accidentally?  It seems relevant, but maybe it was pushed somewhere else and I missed that.

Reply:
@luis-henrix actually should have had a comment attached, again, my mistake. I was going to ask if this should be replaced in favour of the Kernel RBD and CephFS Client chap, or whether or not they should be combined?